layersvt/CMakeLists.txt

```diff
@@ -62,18 +77,31 @@ else()
     add_library(VkLayer_${target} SHARED ${ARGN})
     target_link_Libraries(VkLayer_${target} ${VkLayer_utils_LIBRARY})
     add_dependencies(VkLayer_${target} generate_api_cpp generate_api_h generate_api_html_h)
-    set_target_properties(VkLayer_${target} PROPERTIES LINK_FLAGS "-Wl,-Bsymbolic")
+    if (APPLE)
+        set_target_properties(VkLayer_${target} PROPERTIES LINK_FLAGS "-Wl")
+    else ()
```

Comment:
Did a little more reading on -Wl and it is meaningless without the option that follows

> -Wl,option
Pass option as an option to the linker. If option contains commas, it is split into multiple options at the commas. You can use this syntax to pass an argument to the option. For example, -Wl,-Map,output.map passes -Map output.map to the linker. When using the GNU linker, you can also get the same effect with -Wl,-Map=output.map.

So I think this can just be
```
    if (NOT APPLE)
        set_target_properties(VkLayer_${target} PROPERTIES LINK_FLAGS "-Wl,-Bsymbolic")
    else ()